.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: minimal
```

Comment:
Why don't use standard [way](https://docs.travis-ci.com/user/deployment/npm/) to deploy ?

Current approach will fail every time when it try to publish already existing in npm version.

See some example of deploying to npm: https://github.com/mixbytes/node-eosjs-signature-provider/blob/master/.travis.yml